workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -74,6 +77,7 @@ namespace <#= qualifiedNamespace #>
                     return;
                 }
 
+                <#= profilingStart #>
```

Comment:
How do these look in the profiler? Should it be `Profiler.BeginSample($"{componentname}{methodname}")` instead of only comp name for example?

Reply:
I already have a marker for the method name before this marker. So it will look like `OnComponentUpdate -> TransformInternal`

Reply:
This way things get grouped by method, and not by component

Reply:
I see, sounds good. Can you show a screenshot on what these look like too please?